src/components/quickTransfer/QuickTransfer.jsx

```diff
@@ -0,0 +1,70 @@
+import './quickTransfer.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowRight } from '@fortawesome/free-solid-svg-icons';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Navigation } from 'swiper';
+import 'swiper/css';
+import 'swiper/css/navigation';
```

Comment:
as mentioned in the docs, please group the imports

Reply:
https://github.com/appwex/react-ui-components/blob/master/docs/Code_Contribute_Guidelines.md